mteb/cli.py

```diff
@@ -301,6 +302,56 @@ def potentially_add_cqadupstack_to_results(results: list[mteb.MTEBResults]) -> N
     results.append(result)
 
 
+def merge_yamls(
+    yaml_dict: Dict[str, Any], existing_readme: str
+) -> Tuple[Dict[str, Any], str]:
+    if not existing_readme.endswith("md"):
+        return yaml_dict, ""
+
+    with open(existing_readme) as f:
```

Comment:
What happens if the file does not have an frontmatter? Should we add in an error for that